### PR TITLE
Added support for pluginfactsource, required for SRV and 3.7

### PR DESCRIPTION
**Pull request by zdenekjanda**
Needed for supporting SRV and 3.7. More at https://tickets.puppetlabs.com/si/jira.issueviews:issue-html/PUP-1035/PUP-1035.html

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -28,6 +28,7 @@
   $srv_domain          = $::domain
   # lint:ignore:puppet_url_without_modules
   $pluginsource        = 'puppet:///plugins'
+  $pluginfactsource    = 'puppet:///pluginfacts'
```
Comment (domcleal) [inline manifests/params.pp]: Yeah, I'm OK with this, it'll go into a new minor version at least - people can override the parameter if necessary to get back to the old value.

**Comment by zdenekjanda** [comment]
@mmoll like this in init.pp ok ?

**Comment by ekohl** [comment]
@zdenekjanda that should be good.

**Comment by mmoll** [comment]
looking fine, please squash the commits together into  one and we're good to go here :+1:

**Comment by zdenekjanda** [comment]
Succeed by #212

**Comment by mmoll** [comment]
that could have been done by rebase and force-pushing in this PR but anyway... ;)

**Comment by zdenekjanda** [comment]
oh, thats new, thought that cant force-push rebased head back to github... thx for idea !

**Comment by mmoll** [comment]
We do this the whole time in PRs, just remember not to force-push into "real" repos, which get cloned/pulled by other people. :)

**Pull request closed without merging**